sys-process/daemontools/daemontools-0.76-r11.ebuild

```diff
@@ -0,0 +1,54 @@
+# Copyright 1999-2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit flag-o-matic toolchain-funcs
+
+DESCRIPTION="Collection of tools for managing UNIX services"
+HOMEPAGE="https://cr.yp.to/daemontools.html"
+SRC_URI="
+	https://cr.yp.to/daemontools/${P}.tar.gz
+	http://smarden.org/pape/djb/manpages/${P}-man-20020131.tar.gz"
+S="${WORKDIR}/admin/${P}/src"
+
+LICENSE="public-domain GPL-2"	# GPL-2 for init script
+SLOT="0"
+KEYWORDS="~alpha ~amd64 ~arm ~arm64 ~hppa ~ia64 ~loong ~m68k ~mips ~ppc ~ppc64 ~riscv ~s390 ~sparc ~x86"
+IUSE="selinux static"
+
+RDEPEND="selinux? ( sec-policy/selinux-daemontools )"
+
+PATCHES=(
+	"${FILESDIR}"/${PV}-errno.patch
+	"${FILESDIR}"/${PV}-C99-decls.patch
+	"${FILESDIR}"/${PV}-makefile.patch
+	"${FILESDIR}"/${PV}-implicit-func-decl-clang16.patch
+)
+
+src_configure() {
+	tc-export AR CC
+	use static && append-ldflags -static
+
+	touch home || die
+}
+
+src_install() {
+	keepdir /service
```

Comment:
Huh o.o

Reply:
Should that be removed?

Reply:
Just a very weird directory structure. If it's required by the program to work, then this is correct.